core/modules/comment/comment.module

```diff
@@ -214,7 +214,7 @@ function comment_menu() {
     'page callback' => 'comment_edit_page',
     'page arguments' => array(1),
     'access callback' => 'comment_access',
-    'access arguments' => array('edit', 1),
+    'access arguments' => array('update', 1),
```

Comment:
This looks like an API change. We may need a backwards compatibility addition to make this keep working with the old arguments until Backdrop 2.x.

Reply:
Fixed in https://github.com/backdrop/backdrop/pull/2451